### PR TITLE
[processing] Fix exception in modeler when showing advanced params

### PR DESCRIPTION
**Pull request by luipir**
## Description
fix exception in modeler showing advanced params when wrapper is not a widget

fix promoted inside GeoMove [1] project for Cartolab [2] (A Coruña university)

[1] http://cartolab.udc.es/geomove/
[2] http://cartolab.udc.es

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

- [x] Commit messages are descriptive and explain the rationale for changes
- [ ] Commits which fix bugs include `fixes #11111` in the commit message next to the description
- [ ] Commits which add new features are tagged with `[FEATURE]` in the commit message
- [ ] Commits which change the UI or existing user workflows are tagged with `[needs-docs]` in the commit message and contain sufficient information in the commit message to be documented
- [x] I have read the [QGIS Coding Standards](https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html) and this PR complies with them
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] New unit tests have been added for core changes
- [x] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTE.md#contributing-to-qgis) before each commit

### CODE REVIEW

File: python/plugins/processing/modeler/ModelerParametersDialog.py [hunk 1]
```diff
@@ -231,8 +231,10 @@ def showAdvancedParametersClicked(self):
             self.advancedButton.setText(self.tr('Show advanced parameters'))
         for param in self._alg.parameterDefinitions():
             if param.flags() & QgsProcessingParameterDefinition.FlagAdvanced:
-                self.wrappers[param.name()].widget.setVisible(self.showAdvanced)
-                self.wrappers[param.name()].label.setVisible(self.showAdvanced)
+                wrapper = self.wrappers[param.name()]
+                if issubclass(wrapper.__class__, QgsProcessingModelerParameterWidget):
```
Comment (nyalldawson) [inline python/plugins/processing/modeler/ModelerParametersDialog.py]: Shouldn't this be `if not issubclass....`? QgsProcessingModelerParameterWidget doesn't have .widget/.label

Comment (nyalldawson) [reply python/plugins/processing/modeler/ModelerParametersDialog.py]: Thanks! Let me know if I've missed an API break and I'll fix.

Comment (luipir) [reply python/plugins/processing/modeler/ModelerParametersDialog.py]: no... happen for boolean widget wrapper for grass algorithms (in my case r.null)

Comment (luipir) [reply python/plugins/processing/modeler/ModelerParametersDialog.py]: hmmmm but BooleanWidgetWrapper creates the widget...

Comment (luipir) [reply python/plugins/processing/modeler/ModelerParametersDialog.py]: replicable creating a model with r.null grass algorithm.

**Comment by luipir** [comment]
the issue is replicabel adding ANY grass algorithm in a model ans trying to show advanced parameters
steps:
1) create model
2) add a grass (in my case raster) alg
3) right click in the algorithm (inside the model) to open algorithym widget
4) show advanced params => exception

**Comment by nyalldawson** [comment]
Superseded by #7830

**Comment by nyalldawson** [comment]
Oops #7857

**Pull request closed without merging**